packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1166,17 +1166,16 @@ export function purposeServiceBuilder(
     },
     async clonePurpose({
       purposeId,
-      organizationId,
       seed,
-      correlationId,
-      logger,
+      ctx,
     }: {
       purposeId: PurposeId;
-      organizationId: TenantId;
       seed: purposeApi.PurposeCloneSeed;
-      correlationId: CorrelationId;
-      logger: Logger;
+      ctx: WithLogger<AppContext>;
     }): Promise<{ purpose: Purpose; isRiskAnalysisValid: boolean }> {
+      const { correlationId, authData, logger } = ctx;
```

Comment:
```suggestion
      { authData, correlationId, logger }: WithLogger<AppContext>
    }): Promise<{ purpose: Purpose; isRiskAnalysisValid: boolean }> {
```
We could destructure it like this like we do in other routes